Tests/AssemblerLib/LocalToGlobalIndexMap.cpp

```diff
@@ -29,16 +30,18 @@ class AssemblerLibLocalToGlobalIndexMapTest : public ::testing::Test
         nodesSubset = new MeshLib::MeshSubset(*mesh, &mesh->getNodes());
 
         // Add two components both based on the same nodesSubset.
-        components.emplace_back(new MeshLib::MeshSubsets(nodesSubset));
-        components.emplace_back(new MeshLib::MeshSubsets(nodesSubset));
+        components.emplace_back(std::unique_ptr<MeshLib::MeshSubsets>{
```

Comment:
I guess in `emplace_back` the part `std::unique_ptr<MeshLib::MeshSubsets>{` can be omitted.